package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@openstfoundation/mosaic.js",
-  "version": "0.1.0",
-  "description": "NPM package that provides mosaic contract abis and bins. This package also helps in mosaic chain setup.",
+  "version": "0.10.0",
+  "description": "NPM package that provides mosaic contract abis and bins. This package also helps in mosaic chain setup and interaction with mosaic contracts.",
```

Comment:
This package is not supposed to help with chain setup. Only with Gateway setup. If I remember correctly.

@benjaminbollen ?

Reply:
Do we have a explanation somewhere that makes a clear distinction between "chain setup" and "gateway setup"? That the class is called `ChainSetup` in the package here probably doesn't help...

Reply:
Maybe go with the a bit vague `helps in mosaic setup` for now? 😬

Reply:
Was discussed with @benjaminbollen.
This repo should not have a "do whole chain setup" thing. Instead, it should supply individual actions like "deploy gateway" and possibly even "deploy anchor". Side-chain setup with staked validators however will be done from the mosaic-contracts layer. mosaic.js supplies functionality to token owners (e.g. BenCoin BCO) who want to deploy gateways for their token onto an existing mosaic chain.

Reply:
Long story short: I would not put "mosaic setup" in the description. The description would be that this package helps you to interact with mosaic related contracts. E.g. EIP20Gateway, etc.

Reply:
And with interaction and deployment.

Reply:
Which is not setup (think new side chain with validators setup from mosaic-contracts).